<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="0" rowspan="10" colspan="9">
     <widget class="QWidget" name="widget_2" native="true">
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>2</width>
         <height>2</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QRadioButton" name="spline_display_radio_button">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Сплайн</string>
      </property>
     </widget>
    </item>
    <item row="17" column="11">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="16" column="12">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="16" column="10">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Относ.отклонение</string>
      </property>
     </widget>
    </item>
    <item row="10" column="11">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="16" column="11">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="12" column="11">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7" colspan="2">
     <widget class="QLabel" name="label_29">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
    </item>
    <item row="9" column="11">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="5" colspan="3">
     <widget class="QLabel" name="label">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
    </item>
    <item row="11" column="10">
     <widget class="QLabel" name="label_9">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Rp = </string>
      </property>
     </widget>
    </item>
    <item row="15" column="10">
     <widget class="QLabel" name="label_22">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>icor =</string>
      </property>
     </widget>
    </item>
    <item row="11" column="12">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Ω*см^2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="5">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="maximumSize">
       <size>
        <width>142</width>
        <height>20</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>34.54</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7" colspan="3">
     <widget class="QPushButton" name="cancel_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>23</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Сброс</string>
      </property>
     </widget>
    </item>
    <item row="11" column="11">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="11">
     <widget class="QLabel" name="label_33">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="10">
     <widget class="QLabel" name="label_32">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Вы работаете с файлом</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="5">
     <widget class="QLineEdit" name="lineEdit">
      <property name="maximumSize">
       <size>
        <width>142</width>
        <height>20</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="3" column="9" colspan="5">
     <widget class="QLabel" name="label_2">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Колво узлов сплайна  [5...100]]</string>
      </property>
     </widget>
    </item>
    <item row="6" column="9" colspan="3">
     <widget class="QLabel" name="label_30">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Интервал дифференцирования, mV  [1...10]</string>
      </property>
     </widget>
    </item>
    <item row="14" column="10">
     <widget class="QLabel" name="label_19">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>B =</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="calculate_button">
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Расчет</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="4">
     <widget class="QPushButton" name="spline_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>23</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Сделать сплайн</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4" colspan="5">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="maximumSize">
       <size>
        <width>142</width>
        <height>20</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="15" column="11">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>(0)</string>
      </property>
     </widget>
    </item>
    <item row="13" column="10">
     <widget class="QLabel" name="label_14">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>bc =</string>
      </property>
     </widget>
    </item>
    <item row="10" column="12">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>mV</string>
      </property>
     </widget>
    </item>
    <item row="13" column="12">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>mV</string>
      </property>
     </widget>
    </item>
    <item row="12" column="10">
     <widget class="QLabel" name="label_12">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>ba = </string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="3">
     <widget class="QPushButton" name="open_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>88</width>
        <height>23</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Открыть файл</string>
      </property>
     </widget>
    </item>
    <item row="14" column="12">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>mv</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="3">
     <widget class="QPushButton" name="cut_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>88</width>
        <height>23</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Выбор</string>
      </property>
     </widget>
    </item>
    <item row="10" column="10">
     <widget class="QLabel" name="label_6">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>13</height>
       </size>
      </property>
      <property name="text">
       <string>Ecor =</string>
      </property>
     </widget>
    </item>
    <item row="2" column="9" colspan="5">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Площадь электрода см^2  [0.01...500.0]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="8" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>225</width>
        <height>200</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>2</width>
         <height>2</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
     </widget>
    </item>
    <item row="15" column="12">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>mkA/см^2</string>
      </property>
     </widget>
    </item>
    <item row="13" column="11">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="14" column="11">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>(0)</string>
      </property>
     </widget>
    </item>
    <item row="12" column="12">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>mV</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="15">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="10">
     <widget class="QPushButton" name="change_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>113</width>
        <height>23</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Изменить площадь</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
